src/science/soilsnow/cbl_stempv.F90

```diff
@@ -47,13 +47,15 @@ SUBROUTINE stempv(dels, canopy, ssnow, soil,heat_cap_lower_limit)
     INTEGER :: j,k
     REAL :: exp_arg
     LOGICAL :: direct2min = .FALSE.
-REAL :: heat_cap_lower_limit(mp,ms)
+    REAL :: heat_cap_lower_limit(mp,ms)    ! best to declare INTENT here - rk4417 - phase2
 
     at = 0.0
     bt = 1.0
     ct = 0.0
     coeff = 0.0
 
+    ssnow%otgg(:,:) = ssnow%tgg  ! FEEDBACK (OK to insert this line as per MMY code?) --rk4417 
```

Comment:
otgg is only used by the GW code, so fine to add this here. It won't impact anything.